roles/yoda_rulesets/templates/rules_uu.cfg.j2

```diff
@@ -63,3 +63,7 @@ token_lifetime                 = '{{ token_lifetime }}'
 enable_tape_archive            = '{{ ["false", "true"][enable_tape_archive|int] }}'
 
 temporary_files                = '{{ temporary_files | join (" ") }}'
+
+{% if external_users %}
+external_users_domain_filter   = '{{ external_users_domain_filter | replace('|', '')}}'
```

Comment:
I think adding an additional space filter would make this template more robust against expected whitespace differences in the input value, e.g.:
```
external_users_domain_filter   = '{{ external_users_domain_filter | replace (' ','') | replace('|', ' ')}}'
```